src/common/News/useFetchStockNews.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+import {
+  NEWS_API_URL,
+  NEWS_API_KEY,
+} from 'react-native-dotenv';
+
+export default (query, page = 1, pageSize = 20) => {
+  const [news, setNews] = useState([]);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const newsUrl = new URL(`${NEWS_API_URL}/everything`);
```

Comment:
if I define it before the method, the lazy loading feature won't work because search params will not be updated and something like this will happen:


![image](https://user-images.githubusercontent.com/11687502/71863206-20b64580-3137-11ea-9dca-d68e96bf293d.png)

Reply:
I meant only the string part (`${NEWS_API_URL}/everything`), but it is not that important.